test/object-store/util/event_loop.cpp

```diff
@@ -31,6 +31,9 @@
 #elif REALM_PLATFORM_APPLE
 #include <realm/util/cf_ptr.hpp>
 #include <CoreFoundation/CoreFoundation.h>
+#elif defined(__EMSCRIPTEN__)
+#include <emscripten.h>
+// TODO: implement event loop for Emscripten
```

Comment:
Should this #error or #warning? I'm not sure what the point of including the header then not using it is.